CHANGELOG

```diff
@@ -1,5 +1,64 @@
+Release 89 Plain of Jars (TBD)
+------------------------------
+Scala Common Enrich: bump scala-iglu-client to 0.5.0 (#3092)
```

Comment:
Could you please group all these issues by project as they grouped in previous releases?